packages/fx-core/src/plugins/resource/aad/plugin.ts

```diff
@@ -191,16 +191,18 @@ export class AadAppForTeamsImpl {
     await config.restoreConfigFromContext(ctx);
 
     const manifest = await AadAppManifestManager.loadAadManifest(ctx);
-    config.oauth2PermissionScopeId = manifest.oauth2Permissions[0]?.id;
 
     await DialogUtils.progress?.start(ProgressDetail.Starting);
     if (manifest.id) {
-      if (!skip) {
+      const existingOauth2PermissionScopeId = ctx.envInfo.config.auth?.accessAsUserScopeId;
+
+      if (!skip || !existingOauth2PermissionScopeId) {
         await DialogUtils.progress?.next(ProgressDetail.GetAadApp);
         config = await AadAppClient.getAadAppUsingManifest(
           telemetryMessage,
           manifest.id,
           config.password,
+          manifest.oauth2Permissions[0]?.id,
```

Comment:
can we make sure oauth2Permissions[0] is access_as_user?

Reply:
Oh, thanks, if user modify our template, oauth2Permissions[0] may be not access_as_user, will udpate this part

Reply:
Updated